app/src/main/AndroidManifest.xml

```diff
@@ -128,6 +128,22 @@
       </intent-filter>
     </activity>
 
+    <activity
+      android:name=".signature.SignatureActivity"
+      android:screenOrientation="portrait">
```

Comment:
I am curious, why did we lock this screen to portrait?

Reply:
> We should support landscape mode for signature signing since smaller width screens might not give enough real estate for a complete signature.

This seemed like a better user behaviour since we weren't handling the states but I get the reason now. Updating it to support both.

Reply:
We can support auto rotation while also disabling state saving and restoration using [this property](https://developer.android.com/reference/android/view/View#attr_android:saveEnabled) on the view.